app/views/user_mailer/send_otp.html.erb

```diff
@@ -0,0 +1 @@
+Your OTP is : <%= @user.current_otp %>
```

Comment:
@ajaystha42 indicate the URL for the Violet Rails app. See example below: 

Your OTP for [Violet](https://violet.restarone.solutions) is:

Reply:
okay